src/Entity/ApiDoc.php

```diff
@@ -302,12 +303,17 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
       ->setDisplayConfigurable('view', FALSE);
 
     $fields['api_product'] = BaseFieldDefinition::create('entity_reference')
-      ->setLabel(t('API Product'))
-      ->setDescription(t('The API Product this API is associated with.'))
+      ->setLabel(t('API product'))
+      ->setDescription(t('The API product this API is associated with.'))
       ->setRevisionable(TRUE)
       ->setSetting('target_type', 'api_product')
-      ->setDisplayConfigurable('form', FALSE)
-      ->setDisplayConfigurable('view', FALSE);
+      ->setDisplayOptions('form', [
```

Comment:
Without the `setDisplayOptions` this field was disabled in the form when enabling the module, which would make it confusing for users to understand when viewing the API product access control setting.

Reply:
OK